pythoncalculator/multiply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```

Comment:
```suggestion
    return x * y
```
This should have been multiplication not addition